src/structures/Bot.ts

```diff
@@ -31,7 +31,8 @@ export class Bot extends EventEmitter {
         this.clientOptions = options.clientOptions || {};
 
         this.messages = {
-            missingPermissions: "You don't have the right permissions to run this command!",
+            memberMissingPermissions: "You don't have the right permissions to run this command!",
+            clientMissingPermissions: "I don't have the right permissions to run this command!",
```

Comment:
It would be more consistent to use `bot` everywhere.
```suggestion
            botMissingPermissions: "I don't have the right permissions to run this command!",
```